tasks/configure.yml

```diff
@@ -78,8 +83,26 @@
 - name: Flush handlers to ensure Zookeeper is up to date
   meta: flush_handlers
 
-- name: Wait for Zookeeper port
+- name: Wait for Zookeeper client port
   wait_for:
     port: "{{ zookeeper.client_port }}"
     state: started
-    timeout: 30
+    timeout: 60
+
+- name: Copy zookeeper check script
+  become: yes
+  become_user: "{{ zookeeper.user }}"
```

Comment:
http://docs.ansible.com/ansible/become.html#becoming-an-unprivileged-user - zookeper is unprivileged user